libs/api-client-tiger/scripts/generate.js

```diff
@@ -58,8 +58,8 @@ const downloadAndGenerate = async (specMeta, outputDir, outputFile) => {
 const main = async () => {
     const {
         baseUrl = process.env.BASE_URL,
-        username = process.env.USERNAME,
-        password = process.env.PASSWORD,
+        username = process.env.GD_USER,
+        password = process.env.GD_PASSWORD,
```

Comment:
Please update tiger client README.md as well, the old env vars are used there in examples/snippets.

Reply:
fixed